packages/react/runtime/vitest.config.ts

```diff
@@ -60,6 +60,7 @@ export default defineConfig({
   ],
   resolve: {
     alias: {
+      '@lynx-js/react/compat': path.resolve(__dirname, './compat/index.js'),
```

Comment:
We may also need an alias in `@lynx-js/react-alias-rsbuild-plugin`. Make sure it works with thd old version of `@lynx-js/react` which does not have `./compat` in `exports`